tests/integration/components/frost-icon-test.js

```diff
@@ -10,21 +18,47 @@ describeComponent(
   },
 
   function () {
-    it('renders', function () {
-      // Set any properties with this.set('myProperty', 'value');
-      // Handle any actions with this.on('myAction', function(val) { ... });
-      // Template block usage:
-      // this.render(hbs`
-      //   {{#frost-icon}}
-      //     template content
-      //   {{/frost-icon}}
-      // `);
+    beforeEach(function () {
+      initialize()
+    })
+
+    it('renders default values', function () {
```

Comment:
The idea with this test is that it is the default state of the component with only the required attributes set and then we validate that the things that should be there are actually present. In this case, the class gets set as part of that default.